benchmark/benchmark_control.py

```diff
@@ -0,0 +1,49 @@
+import sys
```

Comment:
Please add module docstrings, here & in all other added files. Don't have to be anything fancy.